examples/RandConfigurationTest.php

```diff
@@ -10,7 +10,20 @@ class RandConfigurationTest extends PHPUnit_Framework_TestCase
     public function testUsingTheDefaultRandFunction()
     {
         $this
-            ->withRand('rand')
+            ->withRand('mt_rand')
```

Comment:
The default rand function would be `rand` rather than `mt_rand` (which is available as an option). Or did `mt_rand` slip in as a default somewhere?

Reply:
I slipped it in by accident, will fix that.